tasks/main.yml

```diff
@@ -1,19 +1,18 @@
 ---
-- include: node_dependencies.yml
+- include: install.yml
+  when: run_mode == 'Install' or run_mode == 'Deploy'
 
-- name: Install Vertica
-  apt:  deb="{{vertica_package_location}}" state=installed
-  sudo: true
+- include: configure.yml
+  when: run_mode == 'Configure' or run_mode == 'Deploy'
 
-- include: node_users.yml
-- include: node_setup.yml
+- include: enable.yml
+  when: run_mode == 'Deploy'
```

Comment:
Deploy should be the same as Install, then Configure, then Enable. So I believe this should be:
when: run_mode == 'Configure' or run_mode == 'Deploy'